addons/website/static/tests/tours/website_edit_save.js

```diff
@@ -0,0 +1,16 @@
+odoo.define("website.tour.website_edit_save", function (require) {
+"use strict";
+
+const tour = require("web_tour.tour");
+
+tour.register("test_edit_save", {
+    test: true,
+    url: "/test_unchanged",
+}, [{
+    content: "Enter edit mode",
+    trigger: 'a[data-action=edit]',
+}, {
+    content: "Save",
+    trigger: '[data-action="save"]',
+}]);
```

Comment:
It's obviously not that simple.
If you read the [task](https://www.odoo.com/web#id=2567752&action=4043&model=project.task&view_type=form&cids=1&menu_id=4720) I attached to this "test request" + the related [commit](https://github.com/odoo/odoo/commit/6714a327368d6fac608149ce098be3e00e725e7c), it seems to only be about:
- drag & dropping some snippets
- on views containing the "oe_structure_id" "hack"
which would then COW the base view + create an inherit view for the oe_structure_id while it should not have cowed the base view